src/main/fc/rc.c

```diff
@@ -354,6 +355,17 @@ FAST_CODE_NOINLINE void rcSmoothingSetFilterCutoffs(rcSmoothingFilter_t *smoothi
                 }
             }
         }
+#if defined(USE_ACC)
+        if (flightModeFlags == ANGLE_MODE || flightModeFlags == HORIZON_MODE) {
+            for (int i = FD_ROLL; i < FD_YAW; i++) {
+                if (!smoothingData->filterInitialized) {
+                    pt3FilterInit(&smoothingData->filterDeflection[i], pt3FilterGain(smoothingData->setpointCutoffFrequency, dT));
+                } else {
+                    pt3FilterUpdateCutoff(&smoothingData->filterDeflection[i], pt3FilterGain(smoothingData->setpointCutoffFrequency, dT));
+                }
+            }
+        }
+#endif
```

Comment:
Same thing with the bit mask check here (see 2nd comment down below). Also renamed `i` to `axis` for clarity:
```c
#if defined(USE_ACC)
        if (FLIGHT_MODE(ANGLE_MODE) || FLIGHT_MODE(HORIZON_MODE)) {
            for (int axis = FD_ROLL; axis < FD_YAW; i++) {
                if (!smoothingData->filterInitialized) {
                    pt3FilterInit(&smoothingData->filterDeflection[axis], pt3FilterGain(smoothingData->setpointCutoffFrequency, dT));
                } else {
                    pt3FilterUpdateCutoff(&smoothingData->filterDeflection[axis], pt3FilterGain(smoothingData->setpointCutoffFrequency, dT));
                }
            }
        }
#endif
```

Reply:
Thank you!  Sorry I stuffed up the flight_mode bit mask.  Very grateful for your advice.
Should be resolved with https://github.com/betaflight/betaflight/pull/10774/commits/a076f8d3bd06c465871c0432753ce633078387c4